### DISCUSSION TITLE
[Resolved] Codename for Fable next (4)?

### DISCUSSION THREAD

**Discussion by alfonsogarciacaro** [fable-compiler/Fable#2461] [Ideas]
**RESOLVED**: Codename for Fable 4 will be "Beyond".

Fable 1: Narumi
Fable 2: Takumi
Fable 3: Nagareyama
Fable 4: ???

Any ideas? It doesn't need to be Japan-related :) A candidate for the multi-language goal would be "Babel", but we can probably discard it for obvious reasons.

> Fable 2 didn't really have a codename, but I've assigned it _a posteriori_ in my mind ;)

**Comment by ncave** [comment]
Here are some non-serious suggestions (I agree your candidate makes sense):
- Fable 4: Babylon  -  (the Greek version sounds a bit better).
- Fable 4: Tokyo Drift  -  (or just Tokyo).
